frontend/main/src/main.cpp

```diff
@@ -223,9 +225,13 @@ int main(const int argc, const char** argv) {
 
             imagepresentation_service.RenderNextFrame(); // executes graph views, those digest input events like keyboard/mouse, then render
 
+            /// XXX temporary fix to make sure that everything that happens post-draw ends up in default window fbo...
+            glBindFramebuffer(GL_FRAMEBUFFER, 0);
```

Comment:
Where is the right place to actually fix this? Image Presentation?

Reply:
I guess so, but might be needed in several places in the frontend that at the moment assume the OpenGL framebuffer state to be in default after returning from graph exec